examples/guides/mp-restful-webservice/src/main/docs/index.adoc

```diff
@@ -0,0 +1,487 @@
+
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+:adoc-dir: {guides-dir}/mp-restful-webservice/src/main/docs
+:java-base: {adoc-dir}/../java/io/helidon/guides/mp/restfulwebservice
+:greet-app: {java-base}/GreetApplication.java
+:main-class: {java-base}/Main.java
+:greet-resource-class: {java-base}/GreetResource.java
+:greeting-message-class: {java-base}/GreetingMessage.java
+:greet-application-class: {java-base}/GreetApplication.java
+:health-resource-class: {java-base}/HealthResource.java
+:pom: {adoc-dir}/../../../pom.xml
+:meta-inf-dir: {adoc-dir}/../resources/META-INF
+:beans-xml: {meta-inf-dir}/beans.xml
+:config-file: {meta-inf-dir}/microprofile-config.properties
+:logging-properties-file: {adoc-dir}/../resources/logging.properties
+:se-guide-adoc: {guides-dir}/se-restful-webservice/src/main/docs/index.adoc
+
+
+= The MP RESTful Web Service Guide
+:description: Helidon MP guide restful web service
+:keywords: helidon, guide, example, mp
+:toc:
+
+Create and build a JAX-RS RESTful web service as your first Helidon MP application.
+
+== What You Will Learn
+You'll learn how to use Helidon MP quickly to create a JAX-RS RESTful web service that accepts these HTTP requests:
+
+|===
+|Method and URL | Result
+
+|`GET localhost:8080/greet` |Returns a generic but friendly greeting
+|`GET localhost:8080/greet/Joe` |Returns a personalized greeting for the specified person
+|`PUT localhost:8080/greet/greeting/Hola` |Changes the greeting used in subsequent responses
+|===
+
+We create the app in these main steps:
+
+. Write a basic Helidon MP app to respond to the HTTP requests.
+. Add code to perform simple health checks.
+. Add code to record simple metrics.
+
+(See the last section to find out how to get the entire finished code for this example.)
+
+== What You Need
+
+|===
+|About 15 minutes
+|An IDE or text editor
+|JDK 1.8 or later
+|Maven 3.5 or later
+|===
+
+//Optional:
+//|===
+//|Docker 18.02 (use the Edge channel to run Kubernetes on your desktop)
+//|`kubectl` 1.7.4
+//|===
+
+== Create Your Application
```

Comment:
I suggest the title "Develop Your Application".

Reply:
Done